kdbg/debugger.cpp

```diff
@@ -118,6 +118,7 @@ const char GeneralGroup[] = "General";
 const char DebuggerCmdStr[] = "DebuggerCmdStr";
 const char TTYLevelEntry[] = "TTYLevel";
 const char KDebugger::DriverNameEntry[] = "DriverName";
+const char ThisFlavor[] = "ThisFlavor";
```

Comment:
What is a "ThisFlavor"? ;) The setting that is stored under this tag is a disassembly flavor, so the correct tag name is "DisassemblyFlavor", right? It should be the exact same tag name that you also use for the global setting.

Reply:
That's easily done, no problem.